src/wapm.spec.ts

```diff
@@ -55,42 +56,87 @@ test.serial('registry url', async ctx =>
   });
 });
 
-test('will login', async ctx =>
+if(userToken) {
+
+  test('will login with token', async ctx =>
 {
-  //Logout
-  logout();
+  // Logout
+  // logout();
 
   //Assert that it will not throw an error
   await ctx.notThrowsAsync(async () =>
   {
     const client = getClient(registry);
     //Login
-    await login(client, username, password);
+    await tokenLogin(client, userToken);
   });
 
   //Assert that we're logged in
   ctx.true(await loggedIn());
-
+  
   //Logout
-  logout();
+  // logout();
 });
 
-test('will publish', async ctx =>
-{
-  //Login
-  const client = getClient(registry);
-  await login(client, username, password);
-
-  //Get the fixture
-  const fixture = resolveFixture();
-
-  //Assert that it will not throw an error
-  await ctx.notThrowsAsync(async () =>
+  test('will publish with token', async ctx =>
   {
-    //Publish (Dry run)
-    await publish(fixture, true);
+    //Login
+    const client = getClient(registry);
+    await tokenLogin(client, userToken);
+  
+    //Get the fixture
+    const fixture = resolveFixture();
```

Comment:
Perhaps combine all `resolveFixture` calls by calling it before individual test cases?